server/game/cards/06-WoE/Muster.js

```diff
@@ -8,14 +8,14 @@ class Muster extends Card {
                 ability.actions.makeTokenCreature(),
                 ability.actions.conditional({
                     condition: (context) =>
-                        context.player.opponent &&
+                        !!context.player.opponent &&
                         context.player.opponent.amber > context.player.amber,
                     trueGameAction: ability.actions.archive()
                 })
             ]),
             effect: 'make a token creature{1}',
             effectArgs: (context) =>
-                context.player.opponent && context.player.opponent.amber > context.player.amber
+                !!context.player.opponent && context.player.opponent.amber > context.player.amber
```

Comment:
This isn't really necessary. The truthy check is fine. If you really want to clean it up you can write it as `context.player.opponent?.amber > context.player.amber` but I'm not sure it gains much

Reply:
Thanks.  I knew they weren't necessary but thought it was better style.  But I don't know anything about JS style so thanks for the info.  Done!